rosdep/python.yaml

```diff
@@ -5350,6 +5350,14 @@ python3-can:
   ubuntu:
     '*': [python3-can]
     xenial: null
+python3-catkin-lint:
+  debian:
+    '*': [python3-catkin-lint]
+    buster: null
+    jessie: null
+    stretch: null
+  fedora: [python3-catkin_lint]
+  ubuntu: [python3-catkin-lint]
```

Comment:
I'm sorry for the back-and-forth, but for some reason I missed this last time.  This also won't work for some of our supported Ubuntu distributions, so we also need to do something like:

```
ubuntu:
  '*': [python3-catkin-lint]
  bionic: null
  xenial: null
```